puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/nightly_packaging.groovy

```diff
@@ -0,0 +1,36 @@
+def setup_nightly_build_environment(args) {
+    def commit_hash = ''
+    def package_name = args.package_name
```

Comment:
I'm not too happy about package_name. For the examples you have here they are the same for every distro, but if we start building nightlies for plugins, then they are suddenly different. Perhaps `project_name` would be more accurate?

Reply:
I chose `package_name` because it is the name of the package that we're asking obal to build.

It differs for `foreman-proxy`. The repo/project is titled `smart-proxy` whereas the package built by obal is titled `foreman-proxy`.

Or maybe I'm not entirely understanding why you aren't happy with the variable name.

Reply:
If you have a plugin it's `rubygem-foreman_memcache` for RPMs but `ruby-foreman-memcache` on Debian. Now I know obal doesn't handle debs right now, but if it's aligning to obal, perhaps `obal_name` would be clearer?

On a side note: longer term I would like obal to understand git repos and how to build archives as well so you can locally point it to a git checkout and let it build a package like you can do with the current `koji-*-jenkins` releaser.

Reply:
`project_name` does seem clearer for the current state of `foreman-infra` nightly building. Down the road I'm okay with renaming things to reflect concepts once `obal` can do more.